pkg/apiserver/api_installer.go

```diff
@@ -108,6 +108,7 @@ func (a *APIInstaller) NewWebService() *restful.WebService {
 	mediaTypes, streamMediaTypes := mediaTypesForSerializer(a.group.Serializer)
 	ws.Produces(append(mediaTypes, streamMediaTypes...)...)
 	ws.ApiVersion(a.group.GroupVersion.String())
+	ws.SetDynamicRoutes(true)
```

Comment:
Given our failed TPR integration test for example, if we create two TPR schemas: `foo.company.com` and `bar.company.com`, both with `v1`, thirdparty server will create a `WebService`, but if we delete `bar.company.com`, thirdparty server does nothing. And of course we cannot remove the entire web service, but we can remove `bar.company.com` related routes, as for go-restful, if we don't enable dynamic routes, we are note allowed to remove these routes.